packages/flutter/lib/src/material/material_app.dart

```diff
@@ -88,6 +95,15 @@ class _MaterialAppState extends State<MaterialApp> implements BindingObserver {
 
   void didChangeSize(Size size) => setState(() { _size = size; });
 
+  void didChangeLocale(ui.Locale locale) => _updateLocaleState(locale);
+
+  void _updateLocaleState(ui.Locale locale) {
```

Comment:
`s/_updateLocaleState/_updateLocale/`

The "state" part of the name doesn't mean anything.